web/src/stream_data.ts

```diff
@@ -61,47 +92,47 @@ class BinaryDict {
         }
     }
 
-    get(k) {
-        const res = this.trues.get(k);
+    get(k: number): T {
+        const res = this.trues.get(k.toString());
```

Comment:
Which raises the question: why is `BinaryDict` using case-folding `FoldDict`s to store numeric keys? We should replace them with `Map`s.

Reply:
Yeah I had the same question. Changed `FoldDict` to `Map` in a separate commit.